clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -19108,6 +19108,21 @@ case Builtin::BI__builtin_hlsl_elementwise_isinf: {
         /*ReturnType=*/Op0->getType(), CGM.getHLSLRuntime().getStepIntrinsic(),
         ArrayRef<Value *>{Op0, Op1}, nullptr, "hlsl.step");
   }
+  case Builtin::BI__builtin_hlsl_wave_active_any_true: {
+    IntegerType *Int1Ty =
```

Comment:
I noticed that other variants of `IntTy` were already defined more broadly but not `Int1Ty`. Is this fine to keep here or should it be move to a larger scope?

Reply:
I am not sure what exactly you are referring to with the scope but I think you can just do
`assert(Op->getType()->isIntegerType(1) && ...)` instead of constructing the type to compare.